pkg/didcomm/envelope/api.go

```diff
@@ -26,7 +52,7 @@ type Packager interface {
 	// []byte: The packed message
 	//
 	// error: error
-	PackMessage(envelope *Envelope) ([]byte, error)
+	PackMessage(envelope *packer.Envelope) ([]byte, error)
```

Comment:
There is a package called envelope but this signature has an Envelope defined in packer?

What is the intended purpose of the Envelope package?

Reply:
I could swap the package names - I needed to pull Envelope out of envelope to avoid an import cycle.

Reply:
I renamed `envelope` (containing just the packager) to `packager` and `packer` (previously `crypto`) to `envelope`